DancingGoat/Controllers/ArticlesController.cs

```diff
@@ -35,7 +38,26 @@ public async Task<ActionResult> Show(string urlSlug)
             }
             else
             {
-                return View(response.Items[0]);
+                try
+                {
+                    var article = response.Items[0];
+                    var recommendationClient = new RecommendationClient(ConfigurationManager.AppSettings["RecommendationApiKey"], 5);
+                    var lastMonth = TimeSpan.FromDays(30).Milliseconds;
+
+                    var recommendedArticles = await recommendationClient
+                        .CreateRequest(Request, Response, codename: article.System.Codename, limit: 2, contentType: article.System.Type)
+                        //.WithFilterQuery("\"personas=Barista\" in 'properties'")
+                        //.WithBoosterQuery($"if 'lastupdated' >= now() - {lastMonth} then 2 else 1")
+                        .Execute();
+
+                    var articles = (await client.GetItemsAsync<Article>(new InFilter("system.codename", recommendedArticles.Select(a => a.Codename).ToArray()))).Items;
+                    article.RelatedArticles = articles.Select(a => (object) a);
+                    return View(article);
+                }
+                catch (Exception ex)
```

Comment:
> Severity	Code	Description	Project	File	Line	Suppression State
Warning	CS0168	The variable 'ex' is declared but never used	DancingGoat	C:\Users\petrs\Documents\GitHub\cloud-sample-app-net\DancingGoat\Controllers\ArticlesController.cs	57	Active


Either introduce logging or get rid of the variable.